modules/kibana/variables.tf

```diff
@@ -51,6 +57,17 @@ variable "credstash_get_cmd" {
   description = "Credstash get command with region and table values set."
 }
 
+/**
+ Argument is expected to have this structure:
+  * security_group_id (required)
+  * certificate (required) - ARN of ACM certificate to be used.
+  * deploy_elb (optional, default: false) - if set to true ELB will be deployed.
+  * deploy_elb_internal (optional, default: true) - if set to false ELB will be deployed as an external one.
+  * deploy_elb_cross_zone (optional, default: true) - if set to false cross AZ for ELB will be off.
+  Below are only required if `deploy_elb` is set to `false` or ommited.
+  * arn - ALB ARN
+  * dns_name - host-header for ALB listener rule
+*/
```

Comment:
I would say the same here, as I did about the other block.

Reply:
Same response